<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmCadClienteDialog</class>
 <widget class="QDialog" name="FrmCadClienteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro Cliente</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RG</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CPF</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefone1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefone2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data de Nasc.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edNome">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edRG">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edCPF">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edTelefone1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edTelefone2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edNasc">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edEmail">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btGravar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gravar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btLimpar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCancelar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>16</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>CADASTRO DE CLIENTES</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
